specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-06-01-preview/SourceControls.json

```diff
@@ -700,6 +792,42 @@
           }
         ]
       }
+    },
+    "PullRequest": {
+      "readOnly": true,
+      "description": "Information regarding pull request for protected branches.",
+      "type": "object",
+      "properties": {
+        "url": {
+          "description": "URL of pull request",
+          "type": "string"
+        },
+        "state": {
+          "$ref": "#/definitions/PullRequestState",
+          "description": "State of the pull request",
+          "type": "object"
+        }
+      }
+    },
+    "PullRequestState": {
+      "description": "Status of the pull request.",
+      "enum": [
+        "Open",
+        "Closed"
```

Comment:
Only two states for PR?

Reply:
Yes, as we're not interested in all other states. For us it is either "in flight" or "closed".